src/Driver/DoctrineDriver.php

```diff
@@ -112,11 +112,14 @@ public function acknowledgeMessage($queueName, $receipt)
      */
     public function peekQueue($queueName, $index = 0, $limit = 20)
     {
-        $query = 'SELECT message FROM bernard_messages WHERE queue = ? LIMIT ' . $index . ', ' . $limit;
-
-        $statement = $this->connection->executeQuery($query, array($queueName));
-
-        return $statement->fetchAll(\PDO::FETCH_COLUMN);
+        $parameters = array($queueName, $index, $limit);
+        $types = array('string', 'integer', 'integer');
+
+        return $this
+            ->connection
+            ->executeQuery('SELECT message FROM bernard_messages WHERE queue = ? LIMIT ?, ?', $parameters, $types)
```

Comment:
Yeah but that means mysql setup etc, currently the DoctrineDriver test does not support this :) would be great if you could work on this since you understand how it works.